package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "strimzi-ui",
+  "private": true,
+  "version": "0.0.1",
+  "description": "Strimzi UI provides a way for managing Strimzi and Kafka clusters (+ other components) deployed by it using a graphical user interface.",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "clean": "rm -rf dist/ && rm -rf generated/",
+    "start": "webpack-dev-server --config utils/build/webpack.dev.js",
+    "build": "webpack --config utils/build/webpack.prod.js"
+  },
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/strimzi/strimzi-ui.git"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/strimzi/strimzi-ui/issues"
+  },
+  "homepage": "https://github.com/strimzi/strimzi-ui#readme",
+  "dependencies": {
+    "@babel/core": "^7.10.5",
```

Comment:
Are any of these dependencies? I'm guessing `express` or equivalent will be in the future - but for now, all of this is just to build assets into the `public` folder, none are needed for running the application.

(Unless we're using the semantics of "it produces production output", but then I'd say they all are dependencies)

Reply:
I havent written it down yet, but I am of the view that it is a production dependency if it produces output that are built. Those listed in dev currently are either developer dependencies, (ie not used in shipped code) or used in the build, but do not generate output.